### ISSUE TITLE
Error: Unable to communicate with server1,server2,server3

### ISSUE THREAD

**Issue by karippery** [ClusterLabs/pcs#253]
surprisingly i got this error.. i can`t find why it showing like this. 

```
root@oreo:~# pcs --debug cluster auth server1 server2 server3 -u hacluster -p 12345678 --force
Running: /usr/bin/ruby -I/usr/share/pcsd/ /usr/share/pcsd/pcsd-cli.rb auth
--Debug Input Start--
{"username": "hacluster", "local": false, "nodes": ["server1", "server2", "server3"], "password": "12345678", "force": true}
--Debug Input End--
Return Value: 0
--Debug Output Start--
{
  "status": "ok",
  "data": {
    "auth_responses": {
      "server1": {
        "status": "noresponse"
      },
      "server2": {
        "status": "noresponse"
      },
      "server3": {
        "status": "noresponse"
      }
    },
    "sync_successful": true,
    "sync_nodes_err": [

    ],
    "sync_responses": {
    }
  },
  "log": [
    "I, [2020-08-10T17:01:35.053030 #7410]  INFO -- : PCSD Debugging enabled\n",
    "D, [2020-08-10T17:01:35.053066 #7410] DEBUG -- : Did not detect RHEL 6\n",
    "I, [2020-08-10T17:01:35.053085 #7410]  INFO -- : Running: /usr/sbin/corosync-cmapctl totem.cluster_name\n",
    "I, [2020-08-10T17:01:35.053098 #7410]  INFO -- : CIB USER: hacluster, groups: \n",
    "D, [2020-08-10T17:01:35.066560 #7410] DEBUG -- : [\"totem.cluster_name (str) = debian\\n\"]\n",
    "D, [2020-08-10T17:01:35.066740 #7410] DEBUG -- : []\n",
    "D, [2020-08-10T17:01:35.066813 #7410] DEBUG -- : Duration: 0.013429973s\n",
    "I, [2020-08-10T17:01:35.066977 #7410]  INFO -- : Return Value: 0\n",
    "I, [2020-08-10T17:01:35.100407 #7410]  INFO -- : No response from: server1 request: /auth, exception: end of file reached\n",
    "I, [2020-08-10T17:01:35.101523 #7410]  INFO -- : No response from: server2 request: /auth, exception: end of file reached\n",
    "I, [2020-08-10T17:01:35.102571 #7410]  INFO -- : No response from: server3 request: /auth, exception: end of file reached\n"
  ]
}
--Debug Output End--

Error: Unable to communicate with server1
Error: Unable to communicate with server2
Error: Unable to communicate with server3

```

status pacemaker is active no error  
status corosync is active no error 
status pscd is active no error   

```
 firewall-cmd --permanent --add-service=high-availability
 firewall-cmd --add-service=high-availability
firewall-cmd --reload
success
```

/etc/corosync/corosync.conf

```
totem {

        cluster_name: mycluster
        transport: udpu
        secauth: off
}
nodelist {
        node {
                ring0_addr: server1
                nodeid: 101
        }
        node {
                ring0_addr: server2
                nodeid: 102
        }
        node {
                ring0_addr: server3
                nodeid: 103
        }

}

quorum {
        # Enable and configure quorum subsystem (default: off)
        # see also corosync.conf.5 and votequorum.5
        provider: corosync_votequorum
        expected_votes: 2
        two_node: 1
        wait_for_all: 1
        last_man_standing: 1
        auto_tie_breaker: 0
}


service {
  name: pacemaker
  ver: 1
}
```
pcs status of server1

 ```
pcs status
Cluster name: mycluster
Stack: corosync
Current DC: oreo (version 1.1.16-94ff4df) - partition WITHOUT quorum
Last updated: Mon Aug 10 17:07:33 2020
Last change: Mon Aug 10 16:34:53 2020 by root via cibadmin on oreo

1 node configured
0 resources configured

Online: [ oreo ]

No resources


Daemon Status:
  corosync: active/enabled
  pacemaker: active/enabled
  pcsd: active/enabled



```
```
gem -v 
3.0.6
ruby -v
2.3.3
```

**Comment by tomjelinek** [comment]
The important part is `No response from: server1 request: /auth, exception: end of file reached` and the same for `server2` and `server3`. Can you connect to pcsd like this `openssl s_client server1:2224`? Can you run `pcs cluster auth server1 server2 server3` and show the part of `/var/log/pcsd/pcsd.log` for the time the command was running?

**Comment by karippery** [comment]
i fix this problem  when  i change gem -v to 2.7.6 and ruby version to 2.3.3 in all my node 

gem update --system 2.7.6
